django/apps/registry.py

```diff
@@ -286,13 +286,14 @@ def get_swappable_settings_name(self, to_string):
         change after Django has loaded the settings, there is no reason to get
         the respective settings attribute over and over again.
         """
+        to_string = to_string.lower()
         for model in self.get_models(include_swapped=True):
             swapped = model._meta.swapped
             # Is this model swapped out for the model given by to_string?
-            if swapped and swapped == to_string:
+            if swapped and swapped.lower() == to_string:
```

Comment:
I wonder if we should always lower case `model._meta.swapped` but this is likely less disruptive and not much of an issue given how `get_swappable_settings_name` is heavily cached.